wow/views.py

```diff
@@ -131,7 +131,10 @@ def address_export(request):
     first_row = addrs[0]
 
     for addr in addrs:
-        addr['ownernames'] = csvutil.stringify_owners(addr['ownernames'])
+        addr["ownernames"] = csvutil.stringify_owners(addr["ownernames"])
```

Comment:
not a huge deal but the rest of this file uses single quotes, so maybe we could continue in that tradition instead of the double quotes here.

Reply:
oh weird! sorry about that, it must've been the Python formatter I was using. Will fix now